test/integration/rekor_monitor_integration_test.go

```diff
@@ -91,6 +92,70 @@ func TestTamperedCheckpoint(t *testing.T) {
 	stopMonitor(t, runCmd)
 }
 
+func TestLogTruncationForking(t *testing.T) {
```

Comment:
It would be clearer to have a separate file for each test case. Please split the test file into multiple ones.